frontend/app/(tabs)/_layout.tsx

```diff
@@ -33,6 +21,7 @@ export default function TabLayout() {
           tabBarIcon: ({ color, size }) => (
             <Entypo name={'home'} color={color} size={size} />
           ),
+          headerShown: false,
```

Comment:
I like this change! Gives more real estate overall. 

However, other developers on the front end are going to have to adjust their pages they are working on. Just let them know in the meeting and get feedback! :)